src/SonarScanner.MSBuild.Common/AnalysisProperties/Property.cs

```diff
@@ -27,51 +27,12 @@
 namespace SonarScanner.MSBuild.Common
 {
     /// <summary>
-    /// Data class to describe an additional analysis configuration property
-    /// /// </summary>
-    /// <remarks>The class is XML-serializable</remarks>
+    /// Data class to describe an additional analysis configuration property.
+    /// </summary>
+    /// <remarks>The class is XML-serializable.</remarks>
     public class Property
     {
-        #region Data
-
-        /// <summary>
-        /// The identifier for the property
-        /// </summary>
-        /// <remarks>Each type </remarks>
-        [XmlAttribute("Name")]
-        public string Id { get; set; }
-
-        /// <summary>
-        /// The value of the property
-        /// </summary>
-        [XmlText]
-        public string Value { get; set; }
-
-        #endregion Data
-
-        #region Public methods
-
-        /// <summary>
-        /// Returns whether the property contains any sensitive data
-        /// </summary>
-        public bool ContainsSensitiveData()
-        {
-            return ProcessRunnerArguments.ContainsSensitiveData(Id) || ProcessRunnerArguments.ContainsSensitiveData(Value);
-        }
-
-        /// <summary>
-        /// Returns the property formatted as a sonar-scanner "-D" argument
-        /// </summary>
-        public string AsSonarScannerArg()
-        {
-            return string.Format(System.Globalization.CultureInfo.InvariantCulture, "-D{0}={1}", Id, Value);
-        }
-
-        #endregion Public methods
-
-        #region Static helper methods
-
-        //TODO: this expression only works for single-line values
+        // ToDo: This expression only works for single-line values
```

Comment:
Can you please create an issue for this?

Reply:
I've deleted the `ToDo` instead. The name `SingleLinePropertyRegEx` is pretty clear about it.

I don't have `Why` for the issue, as I don't know why we would need to support multitline properties and under what scenarios. It has been like this for long time and users didn't complain.